### PR TITLE
Simplify iOS troubleshooting FAQs earlier than 1.5.3

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR resolves the request https://github.com/corona-warn-app/cwa-website/issues/3238 "Remove FAQ information for versions below minimum supported".

The minimum supported versions of the Corona-Warn-App are:

Android | iOS
--- | ---
1.0.4 | 1.5.3

so the following iOS-related FAQ troubleshooting articles are simplified accordingly:

- https://www.coronawarn.app/en/faq/results/#migration_Apple
- https://www.coronawarn.app/de/faq/results/#migration_Apple
- https://www.coronawarn.app/en/faq/results/#app_does_not_open
- https://www.coronawarn.app/de/faq/results/#app_does_not_open

In addition I have added two new variables into [gulpfile.mjs](https://github.com/corona-warn-app/cwa-website/blob/master/gulpfile.mjs):

```
[ios.minimum-app-version]', '1.5.3'
[android.minimum-app-version]', '1.0.4'
```

to assist with consistency in the FAQs.

### CODE REVIEW

File: src/data/faq_de.json [hunk 1]
```diff
@@ -2222,9 +2222,7 @@
...
                                 "active": false,
                                 "textblock": [
-                                    "<b>AKTUELL</b><br/> Dieses Problem wurde mit Version 1.5.0 behoben.<hr/> Wenn Sie Ihre Daten über iTunes auf ein neues Gerät übertragen oder ein Backup wiederhergestellt haben, funktioniert die App nicht richtig. Unter Status (vor Version 2.12 (Startseite) landen Sie in einer Endlosschleife und/oder die Risiko-Ermittlung lässt sich nicht einschalten. Der Risikostatus wird nicht angezeigt.",
-                                    "Das ist leider ein bekanntes Problem mit der Sicherung der Zufalls-IDs, siehe <a href='#backup' target='_blank'>'Werden die gesammelten Zufalls-IDs in meiner Smartphone-Datensicherung (Backup) berücksichtigt?'</a>.",
-                                    "Bitte installieren Sie die Corona-Warn-App auf Ihrem neuen iPhone neu."
+                                    "Bitte installieren Sie die aktuelle Version [ios.current-app-version] der Corona-Warn-App auf Ihrem neuen iPhone. Dieses Problem wurde mit Version 1.5.0 behoben, allerdings ist [ios.minimum-app-version] die früheste unterstützte Version der App."
```
Comment (Ein-Tim) [inline src/data/faq_de.json:2225]: I would suggest to use "die äteste unterstützte Version der App."

File: src/data/faq_de.json [hunk 2]
```diff
@@ -2327,8 +2325,7 @@
                                 "anchor": "app_does_not_open",
                                 "active": false,
                                 "textblock": [
-                                    "<b>AKTUELL</b><br/> Inzwischen steht die App-Version 1.2.1 zur Verfügung, die dieses Problem behebt. Bitte aktualisieren Sie die App. <hr/>",
-                                    "Mit dem Update auf 1.2.0 trat bei vielen Nutzern das Problem auf, dass sie die Corona-Warn-App nicht mehr öffnen konnten."
+                                    "Bitte aktualisieren Sie die App auf die neueste Version [ios.current-app-version]. Das Problem wurde schon mit der App-Version 1.2.1 behoben, allerdings ist [ios.minimum-app-version] die früheste unterstützte Version der App."
```
Comment (Ein-Tim) [inline src/data/faq_de.json:2328]: I would suggest to use "die äteste unterstützte Version der App."

File: src/data/faq_de.json (hunk 1 above)
Comment (Ein-Tim) [inline src/data/faq_de.json:2225]: I would suggest to use "die äteste unterstützte Version der App."

**Review by Ein-Tim — COMMENTED** [review]
As talked about in https://github.com/corona-warn-app/cwa-website/issues/3238. I agree with this & support it. Some minor comments on the German wording.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

I found both "älteste" and "früheste" used in other Internet articles.

See https://support.apple.com/de-de/guide/classroom/apd606b230fc/2.3/mac/11.0 which uses "Früheste unterstützte macOS-Version".

Do you think "früheste" is wrong in this context?

**Comment by brianebeling** [comment]
@Ein-Tim @MikeMcC399 We decided to go with "älteste", as it sounds the most correct to us in this context. It also reinforces that we want users to be on the newest CWA version. Something being old sounds a bit worse than it being early, right? 😆

I will make the change and then its good to go.

Thank you very much!

**Comment by brianebeling** [comment]
I'm starting to think we should reconsider the sentence as a whole. As someone looking at the FAQ and searching for answers to troubleshooting it might confuse the user to read that it was fixed in version X, but version Y is the oldest supported version. It could be beneficial to reduce information given to only include required information, for example: "This has been fixed in version X, please update to the latest version".

**Comment by Ein-Tim** [comment]
@brianebeling 

Or even easier: "This has been fixed in an update. If you encounter this error, please update your app."

**Comment by MikeMcC399** [comment]
@brianebeling / @Ein-Tim 

I agree that it would be good to simplify the information, especially as a user does not get the choice when they go to the App Store for an update - they just see and get the latest released version.

Which suggestion should we take?

1. "This has been fixed in version X, which is no longer supported, please update to the latest version using the AppStore".
2. "This has been fixed in an update. If you encounter this error, please update your app."

**Comment by brianebeling** [comment]
@MikeMcC399 @Ein-Tim Ok. I updated the texts and tests are running currently. Let me know if you have further suggestion. From my side this can be merged.

File: src/data/faq.json [hunk 3]
```diff
@@ -2209,9 +2209,7 @@
                                 "anchor": "migration_Apple",
                                 "active": false,
                                 "textblock": [
-                                    "<b>UPDATE</b><br/>This problem has been solved with version 1.5.0.<hr/> If you transfer your data to a new iPhone using iTunes, or restore a backup, the app does not work properly. On the Status tabs ('Start Screens' before version 2.12), you end up in a loop, or the exposure logging cannot be activated. The risk status is not displayed.",
-                                    "This is a known problem with backing up the random IDs, see also <a href='#backup' target='_blank' >'Will the collected random IDs be included when I backup data on my phone?'</a>.",
-                                    "Please reinstall the Corona-Warn-App on your new iPhone."
+                                    "The problem has been fixed with app version 1.5.0. You can update the Corona Warning app to the latest version [ios.current-app-version] in the <a href='https://apps.apple.com/de/app/corona-warn-app/id1512595757' target='_blank' rel='noopener noreferrer'>Apple App Store</a>."
```
Comment (MikeMcC399) [inline src/data/faq.json:2212]: "Corona Warning app" => "Corona-Warn-App"

This is the name consistently used also in English language texts.

File: src/data/faq.json [hunk 4]
```diff
@@ -2314,8 +2312,7 @@
                                 "anchor": "app_does_not_open",
                                 "active": false,
                                 "textblock": [
-                                    "<b>UPDATE</b><br/> App version 1.2.1 is now available and resolves this issue. Please update your app. <hr/>",
-                                    "Unfortunately, an error was introduced with the latest update (1.2.0): Many users were not able to open the app."
+                                    "The problem has been fixed with app version 1.2.1. You can update the Corona Warning app to the latest version [ios.current-app-version] in the <a href='https://apps.apple.com/de/app/corona-warn-app/id1512595757' target='_blank' rel='noopener noreferrer'>Apple App Store</a>."
```
Comment (MikeMcC399) [inline src/data/faq.json:2315]: Same comment for app name.

**Review by MikeMcC399 — COMMENTED** [review]
Suggest to use Corona-Warn-App also in the English text.

**Comment by brianebeling** [comment]
> Suggest to use Corona-Warn-App also in the English text.

Oh, my bad. I didn't notice the translation error. Thanks!

**Comment by MikeMcC399** [comment]
@MikeMcC399 

I suggest to write CWA exactly as "Corona-Warn-App" and not "Corona-Warn App" for consistency with the rest of the web and with the app stores.

**Comment by MikeMcC399** [comment]
@brianebeling 

GitHub does not allow me to approve a PR which I have authored, so let me do it in text form:

All looks good now!

Thanks (also to @Ein-Tim) for simplifying the text for the benefit of regular users!

**Pull request merged**